config/testgrids/conformance/kind/conformance-kind.yaml

```diff
@@ -0,0 +1,10 @@
+dashboards:
+- name: conformance-kind
```

Comment:
should we be moving all of these?

Reply:
you mean independently in its own folder or to the conformance-all?
I don´t have a strong opinion but I think that it should belong to sig-testing/kind and conformance-all

Reply:
The conformance dashboards seem large enough that they should be split into their own file(s). 

This change doesn't change the contents of the dashboards; it wasn't on `sig-testing`, and it still isn't, but it keeps it on both conformance tabs.

@aojea, if you're looking to add this to sig-testing also, I would copy the tab in question and put it in the sig-testing file as you had before, but no need to delete anything in conformance.

Reply:
I think this is ok, we can follow up if necessary with a new pull request

Reply:
I'm confused that the conformance-kind dashboard here only seems to have a single tab

Reply:
it should have 11 currently

Reply:
@BenTheElder I believe the others tabs are prow annotations, and aren't located in either YAML doc

Reply:
[trying to think why this one is the sole one not in an annotation]

Reply:
While there's a prow job named `ci-kubernetes-kind-conformance-latest-1-14`, I don't think there's one named `ci-kubernetes-kind-conformance-latest-1-14-arm64`

Reply:
derp. this one is from third party CI. TBH I'm slightly thinking we should move it to another dashboard but that's orthogonal do this work.

thanks